components/layout/blog-homepage-section/blog-homepage-section.hbs

```diff
@@ -0,0 +1,168 @@
+<div class="section--bg-white">
+    <div class="content-block--homepage-blog">
+
+        <!-- Featured Blog -->
+        <a href="#" class="content-block--clickable content-block--blog-featured-home">
+
+            <div class="content-block__upper-blog">
+                <picture class="post-image">
+                    <source media="(max-width: 799px)" srcset="https://www.fillmurray.com/g/600/634">
+                    <source media="(min-width: 800px)" srcset="https://www.fillmurray.com/g/525/720">
+                    <img class="content-block__image" src="https://www.fillmurray.com/g/545/747" role="presentation"
+                        alt="" />
+                    <source>
+                </picture>
+            </div><!-- .content-block__upper-blog -->
+
+            <div class="content-block__lower-blog">
+                <h4 class="content-block__title anchor-primary">
```

Comment:
Should be an h3, right, if we're assuming the "From the Blog" is the h2 for the section.

Reply:
Correct. Fixed